helpers/build-specs.sh

```diff
@@ -104,6 +104,16 @@ for spec in "${specs[@]}"; do
     echo "-> $spec"
     if [ -z "${skip_specs[$spec]}" ]; then
         echo "::: $spec"
+
+        builder_spec=0
+        grep -q BUILDER_RPM_VERSION "$spec" || builder_spec=1
+        if [ -n "${BUILDER_EPOCH}" ] && [ ${builder_spec} == 0 ]; then
```

Comment:
`==` is not a valid test in shell and this looks weird, try this:

```suggestion
        if [ -n "${BUILDER_EPOCH}" ] && grep -q BUILDER_RPM_VERSION "$spec"; then
```

Reply:
Hah I actually had something like this originally but it didn't work, probably a typo or something. I really hate shell programming!